src/content/docs/agile-handbook/heroing/what-is-a-hero.mdx

```diff
@@ -81,39 +86,101 @@ Common questions and requests include:
-      - If the writer closes the ticket, they must leave a comment explaining why.
-    - Writer needs to add: 
-      - Acceptance criteria
-      - User story
-      - Action items
-      - Resources 
-    - While rewriting, focus on making the ticket as swarmable as possible
-    - Assign a priority [See priority info in our team's `DOC Jira and Github fields - Project 401 - 2022` doc]
-    - Add the following label: `doc_groomed`
-3. Triaged tickets enter sprints in one of two ways:
-    - **Scrum leaders**: Bring in tickets based on marked priority
-    - **Liaisons**: Bring in high value issues
-    - **Squads**: During backlog grooming the team can chat about bringing in tickets to fill out a sprint
-
 <Callout variant="important">
 For writers in Europe:
 
 - Route incoming issues to the liaison as you have time. As writers in Europe are "eternal heroes" it isn’t fair to have you do this every day.
 - You should still triage issues routed to you from your liaisonships once per day.
  </Callout>
 
+## Customer feedback sidekick [#cf_sidekick]
+
+Feedback is a gift. Every piece of meaningful feedback represents someone taking time out of their day to express something to us. It's important that we honor that gift. The sidekick role was created so that we have a role and a process for getting a human being to read through and think about and sometimes respond to every piece of feedback that people write to us. 
+
+Customer feedback from our docs site creates tickets in Jira. It's the sidekick's responsibility to look at every ticket that comes in (usually about 5–10 per day) and make decisions about each one.
+
+We have three options for these tickets:
+
+1. Close it.
+2. Fix it.
+3. Backlog it.
+
+<Callout variant="important">
+Unlike the hero, we don't take any points out of our sprint for the sidekick work. If you're spending more than 60 minutes per day on customer feedback tickets, that's too much time! Please let your manager know.
+ </Callout>
+
+### Close customer feedback
+
+Many pieces of customer feedback simply aren't actionable. That's ok! Even if we can't act on a specific piece of feedback, we can use this feedback in aggregate to get an overall sense of how our docs site or a specific category of our docs site is landing with our readers.
+
+Here are some fictional examples of feedback we might not be able to take action on specifically:
+
+`These docs are bad.`
+
+or 
+
+`Help!`
+
+There's also feedback that falls into the category of nonsense or mistakes.
+
+Follow this process to close tickets that aren't actionable or don't make sense:
+
+1. Add a category label based on the URL, such as `doc_apm` or `doc_errors-inbox`.
+2. Set the Workflow status to **Closed** 
+3. Set the Resolution to **Cannot Reproduce**, **Filed by Mistake**, or **Won't Do** depending on what feels most appropriate.
+4. Add a brief note about why you closed it. Usually, "Not actionable" is plenty.
+
+### Fix customer feedback
```

Comment:
i think because we have a time limit (no more than >1hr for ALL tickets per day), we should either allot more time to address quick fixes, or come up with a firmer definition of what a quick fix is. For example, do we consider it a "quick fix" even if we have to reach out to a SME? Is it a quick fix only if we don't need to reach out to a SME? I made 1 fix this week as sidekick because it was a "quick message" over slack, then updating the doc itself with the example. That was about ~15 minutes of work that was p easy, but it was still *time*. maybe I'm getting too in the weeds

Reply:
alternative examples: are quick fixes only *language* in nature, or broken links? What about feedback that's on a low traffic doc, but there's a possibility that the content on a doc is flat out wrong or out of date? This will get hammered out I'm sure but these are the questions that i've encountered this wk, so i'm riffing

Reply:
My general sense, based on what I've seen so far, is that needing to reach out to a SME almost never leads to a quick fix. I've added some guidance around this.